### DISCUSSION TITLE
Light Mode

### DISCUSSION THREAD

**Discussion by mimtiyazudd** [f1multiviewer/issue-tracker#360] [Feature requests] [answered]
As I see, there is only dark mode. Can we have a toggle to switch between light and dark mode, please.

**Comment by LapsTimeOFF** [comment, marked as answer]
Added in [v1.32.0](https://multiviewer.app/changelog#version-1.32.0)

**Comment by mimtiyazudd** [comment]
Mega Job guys…
♥️
